ui/src/app/machines/components/MachineHeaderForms/ActionFormWrapper/DeployForm/DeployFormFields/DeployFormFields.tsx

```diff
@@ -184,6 +188,34 @@ export const DeployFormFields = (): JSX.Element => {
                 "u-sv2": userDataVisible,
               })}
             />
+            <FormikField
+              disabled
+              type="checkbox"
+              name="enable_hw_sync"
+              label={
+                <>
+                  Periodically sync hardware{" "}
+                  <Tooltip
+                    positionElementClassName="u-display-inline-important"
+                    message={breakLines(
+                      "Enable this to make MAAS periodically check the hardware configuration of this machine and reflect any possible change after the deployment."
+                    )}
+                  >
+                    <Button
+                      type="button"
+                      appearance="base"
+                      aria-label="more about periodically sync hardware"
+                      className="u-no-margin--bottom"
+                    >
+                      <Icon name="information" />
+                    </Button>
+                  </Tooltip>
+                  {/* TODO: Update docs links https://github.com/canonical-web-and-design/app-tribe/issues/787 */}
+                  <a href="#">Hardware sync docs</a>
```

Comment:
This is wreaking havoc on the console output (it doesn't fully finish starting the ui due to the lint errors) so it might be better to have something like `href="#todo"`.